QuizBowlDiscordScoreTracker/Web/wwwroot/js/main.js

```diff
@@ -1,34 +1,39 @@
 "use strict";
 
-var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
 var nameDisplay = document.getElementById("name");
-var sound1 = document.getElementById("sound1");
-
-var shouldEmitSound = true;
 
 function removeBarrier() {
     document.getElementById("interactBarrier").style.display = "none";
 }
 
-connection.on("PlayerBuzz", function (name) {
-    document.body.style.backgroundColor = "red";
-    nameDisplay.textContent = name;
+if (window.location.search.length <= 1) {
+    nameDisplay.textContent = "No channel specified."
+} else {
+    var connection = new signalR.HubConnectionBuilder().withUrl("/hub").build();
+    var buzzSound = document.getElementById("buzzSound");
+
+    var shouldEmitSound = true;
 
-    if (shouldEmitSound) {
-        sound1.play();
-    }
+    connection.on("PlayerBuzz", function (name) {
+        document.body.style.backgroundColor = "red";
+        nameDisplay.textContent = name;
 
-    shouldEmitSound = false;
-});
+        if (shouldEmitSound) {
+            buzzSound.play();
+        }
 
-connection.on("Clear", function () {
-    document.body.style.backgroundColor = "green";
-    nameDisplay.textContent = "";
-    shouldEmitSound = true;
-});
+        shouldEmitSound = false;
+    });
 
-connection.start().then(function () {
-    connection.invoke("AddToChannel", window.location.search.substring(1));
-}).catch(function (err) {
-    return console.error(err.toString());
-});
\ No newline at end of file
+    connection.on("Clear", function () {
+        document.body.style.backgroundColor = "green";
```

Comment:
We don't have to change this now, but we should consider a color blind friendly palette, since red/green colorblind users may have trouble distnguishing if its buzzed or not. This is a good article on it: https://blog.designbro.com/how-to-design-a-color-blind-friendly-palette/

Reply:
I think it's fine since the presence of text or not also symbolizes a buzz, not to mention the loud noise that's played.